pkgs/by-name/st/stegsnow/package.nix

```
@@ -0,0 +1,55 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "stegsnow";
+  version = "20130616-8";
+
+  src = fetchFromGitLab {
+    domain = "salsa.debian.org";
+    owner = "pkg-security-team";
+    repo = "stegsnow";
+    rev = "debian/${finalAttrs.version}";
+    hash = "sha256-5GmU2HfWSLJ8DwjZZmWW277rJhA73is6606YZAKo4r8=";
+  };
+
+  postPatch = ''
+    for i in $(cat debian/patches/series); do
+      patch -p1 < "debian/patches/$i"
+    done
+
```

this better be done in `prePatch` to use the $patches variable
```suggestion
  prePatch = ''
    for i in $(< debian/patches/series); do
      patches=+("debian/patches/$i")
    done
  '';
  postPatch = ''
```